app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/models/Allergen.java

```diff
@@ -0,0 +1,33 @@
+package com.epfl.neighborfood.neighborfoodandroid.models;
+
+import com.epfl.neighborfood.neighborfoodandroid.R;
+
+public enum Allergen {
+    CELERY (R.drawable.celery, "celery"),
+    MILK (R.drawable.milk, "milk"),
+    FISH (R.drawable.fish, "fish"),
+    CHEESE (R.drawable.cheese, "cheese"),
+    GLUTEN (R.drawable.gluten, "gluten"),
+    HONEY (R.drawable.honey, "honey"),
```

Comment:
We should update the first letters to be capital letters in the labels for all allergens